.vib/ghost/goss/vars.yaml

```diff
@@ -0,0 +1,29 @@
+binaries:
+  - mysql
+  - node
+  - python
+directories:
+  - paths:
+    - /home/ghost
+  - paths:
+    - /opt/bitnami/ghost
```

Comment:
The permissions of `/opt/bitnami/ghost` are explicitly set:

https://github.com/bitnami/containers/blob/6d597b976a0dbb9f10329820ef5f80cb75618d77/bitnami/ghost/5/debian-11/rootfs/opt/bitnami/scripts/ghost/postunpack.sh#L43

Moreover, the check should work:

```bash
$ stat $GHOST_BASE_DIR
  File: /opt/bitnami/ghost
  Size: 4096      	Blocks: 8          IO Block: 4096   directory
Device: 8dh/141d	Inode: 4073330     Links: 6
Access: (0777/drwxrwxrwx)  Uid: (  999/ UNKNOWN)   Gid: (    0/   ghost)
Access: 2023-05-17 14:04:41.896760757 +0000
Modify: 2023-05-17 13:07:34.000000000 +0000
Change: 2023-05-17 14:04:41.807761876 +0000
 Birth: 2023-05-17 14:04:25.346846632 +0000
```